diagnostics/model_accuracy/verify_inference_model.py

```diff
@@ -39,69 +39,71 @@ def download_repo(repo, target_name=None, version='master'):
     command_executor("git clone " + repo)
 
 
-def run_inference(model_name):
-    parameters = '[{"model_type" : "Image Recognition", "model_name" : "Inception_v4","DATASET" : "/mnt/data/TF_ImageNet_latest/","CHECKPOINT" : "/nfs/site/home/skantama/validation/models/research/checkpoints/inception_v4.ckpt"}, {"model_type" : "Image Recognition", "model_name" : "MobileNet_v1","DATASET" : "/mnt/data/TF_ImageNet_latest/","CHECKPOINT" : "/nfs/site/home/skantama/validation/models/research/checkpoints/mobilenet_v1_1.0_224.ckpt"}, {"model_type" : "Image Recognition", "model_name" : "ResNet50_v1","DATASET" : "/mnt/data/TF_ImageNet_latest/","CHECKPOINT" : "/nfs/site/home/skantama/validation/models/research/checkpoints/resnet_v1_50.ckpt"}, {"model_type" : "Object Detection", "model_name" : "SSD-MobileNet_v1", "CHECKPOINT" : "/nfs/site/disks/aipg_trained_dataset/ngraph_tensorflow/fully_trained/ssd_mobilenet_v1_coco_2018_01_28/"}]'
+def run_inference(model_name, models_dir):
+    parameters = \
+    '[{"model_type" : "Image Recognition", "model_name" : "Inception_v4", \
+    "cmd" : "OMP_NUM_THREADS=28 KMP_AFFINITY=granularity=fine,compact,1,0 python eval_image_classifier.py \
+    --alsologtostderr --checkpoint_path=/nfs/site/home/skantama/validation/models/research/checkpoints/inception_v4.ckpt \
```

Comment:
Indent the command fields for better readability.

Reply:
made these changes.